CHANGELOG.md

```diff
@@ -1,4 +1,20 @@
 ## 3.28.0 (Unreleased)
+
+FEATURES:
+* **New Data Source:** `google_redis_instance` ([#6649](https://github.com/terraform-providers/terraform-provider-google/pull/6649))
+* **New Resource:** `google_notebook_environment` ([#6639](https://github.com/terraform-providers/terraform-provider-google/pull/6639))
+* **New Resource:** `google_notebook_instance` ([#6639](https://github.com/terraform-providers/terraform-provider-google/pull/6639))
+
+IMPROVEMENTS:
+* app_engine: added the option to specify `database_type` in `google_app_engine_application` ([#6629](https://github.com/terraform-providers/terraform-provider-google/pull/6629))
```

Comment:
This enables provisioning firestore on a new project-- maybe worth calling that out explicitly? It was a very demanded feature.